system/lib/libc/musl/src/env/__stack_chk_fail.c

```diff
@@ -23,6 +23,9 @@ void __init_ssp(void *entropy)
 
 void __stack_chk_fail(void)
 {
+#if defined(__EMSCRIPTEN__) && !defined(NDEBUG)
```

Comment:
Maybe add a comment about why we have more logging than normal builds?

Reply:
Will do.  I honestly don't know why musl doesn't print anything at all... this is the same message you can find in glibc.